README.md

```diff
@@ -66,6 +67,14 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
 [![](https://img.shields.io/badge/back%20to%20top-%E2%86%A9-blue)](#shortcuts)
 
 
+| Name             | Badge                                                                                                                                   | Markdown                                                                                                                                  |
```

Comment:
This is an orphaned section.
Also, try to keep your PR to a specific thing.

Reply:
actually, it was a blog section